submissions/sprint_b/app.py

```diff
@@ -0,0 +1,23 @@
+from sprint_b import app, jsonify
+from sprint_b.helpers import EpithetGenerator, Vocabulary
+
+
+@app.route('/')
+def generate_epithet():
+    epithet = EpithetGenerator.generate_epithet()
+    return jsonify({"epithet": epithet})
+
+
+@app.route('/epithets/<quantity>')
+def generate_epithets(quantity):
+    quantity = int(quantity)
+    epithets = []
+    for n in range(quantity):
```

Comment:
Your generate method should accept the quantity of epithets to generate 
and return a list of epithets. All of the logic for looping some quantity of 
times should be part of your generate method and not defined in the view. This 
DRYs up the view so you don't have to repeat iteration logic in each view. 

Define an EpithetGenerator class with methods to:
select one random word from each column of the list.
generate a quantity of epithets from a vocabulary file loaded from a path.